website/docs/configuration/snippet-choice.md

```diff
@@ -0,0 +1,187 @@
+The snippets are short pieces of code coming from difference public sources such as GitHub, GitLab, Stackoverflow, ...
+
+Some product such as ScanOSS and FossID scrape those public sources and build a Knowledge Base of snippets, classifying
```

Comment:
"Some product such as" -> "So-called snippet scanners like"

Reply:
I wouldn't capitalize "Knowledge Base" or "Author, Version and License" in the next sentence.

Reply:
Note: I'm pausing review here.